drivers/scheduler/k8s/specs/mysql-backup-psa-restricted/px-mysql-app.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql
+  annotations:
+    startDataSupported: "true"
+    username: "root"
+    password: "password"
+    databaseName: "mysql"
+    port: "3306"
+    appType: "mysql"
+spec:
+  type: NodePort
+  selector:
+    app: mysql
+  ports:
+    - port: 3306
+      targetPort: 3306
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql
+  labels:
+    app: mysql
+spec:
+  selector:
+    matchLabels:
+      app: mysql
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: mysql
+    spec:
+      schedulerName: stork
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
+        fsGroup: 1010
+      containers:
+      - image: mysql:5.6
```

Comment:
@snigdha-px can you try with 8.0 or 8.4 as 5.6 very old version.

Reply:
done